perf/lua/benchmark.lua

```diff
@@ -119,13 +123,37 @@ function M.argparse(arg, argtable, custom_help)
     return params
 end
 
+local function load_average()
+    local path = '/proc/loadavg'
+    local fh = assert(fio.open(path, {'O_RDONLY'}))
+    local loadavg_buf = fh:read(1024)
+    fh:close()
+    -- Format is '0.89 0.66 0.80 2/1576 1203510'.
+    local loadavg_re = '(%d+.%d+) (%d+.%d+) (%d+.%d+)'
+
+    return { loadavg_buf:gmatch(loadavg_re)() }
+end
+
+local function perf_context()
+    local ctx = {}
```

Comment:
I suppose that
```lua
return {
    date = os.date('%Y-%m-%d %H:%M:%S'),
    -- ...
}
```
Looks better. Feel free to ignore.

Reply:
Updated:

```diff
--- a/perf/lua/benchmark.lua
+++ b/perf/lua/benchmark.lua
@@ -135,15 +135,14 @@ local function load_average()
 end
 
 local function perf_context()
-    local ctx = {}
-    ctx.date = os.date('%Y-%m-%d %H:%M:%S')
-    ctx.load_avg = load_average()
-    ctx.tarantool_version = tarantool.version
-    ctx.build_target = tarantool.build.target
-    ctx.build_flags = tarantool.build.flags
-    ctx.host_name = io.popen('hostname'):read()
-
-    return ctx
+    return {
+        build_flags = tarantool.build.flags,
+        build_target = tarantool.build.target,
+        date = os.date('%Y-%m-%d %H:%M:%S'),
+        host_name = io.popen('hostname'):read(),
+        load_avg = load_average(),
+        tarantool_version = tarantool.version,
+    }
 end
 
 function M.new(opts)
```

(keys sorted alphabetically!)